playbooks/roles/ses-rook/files/test_pvc.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
nit: `---` missing

Reply:
somehow `---` is not mandatory if you dont start your yaml with a directive